docs/DeveloperGuide.md

```diff
@@ -220,14 +220,60 @@ Here is an example usage scenario and how `StaticContext` is used in KnottyPlann
 
 ## **Appendix: Planned Enhancements**
 
-1. Prefix enhancements
+**1. Prefix Enhancements**
 
-Currently: The view wedding command in KnottyPlanners does not require a need for a prefix to represent the wedding. Thus, when associating a wedding to a `Person`, it is done by _tagging_ them to it, making the prefix 't/'. 
+**Currently:** The view wedding command in KnottyPlanners does not require a need for a prefix to represent the wedding. Thus, when associating a wedding to a `Person`, it is done by _tagging_ them to it, making the prefix 't/'. 
 
-Plan: To make the association of `Wedding` and `Person` more intuitive, we will change the prefix from 't/' to 'w/', representing the `Wedding` that a `Person` is associated with.
+**Plan:** To make the association of `Wedding` and `Person` more intuitive, we will change the prefix from 't/' to 'w/', representing the `Wedding` that a `Person` is associated with.
+
+**2. Forced Deletion and Clearing**
+
+**Currently:** To prevent accidental deletions and clearing of data in KnottyPlanners, a confirmation will be required from the user to ensure that the deletion or clearing is intentional.
+
+**Plan:** To include a force deletion or clearing command that allows the user to remove the data without KnottyPlanners requiring a confirmation from them.
+
+**3. Improved Filtering**
+
+**Currently:** The filter command in KnottyPlanners only allows for filtering to be done through the name or job fields.
+
+**Plan:** To allow users to filter with other fields such as phone number, address and email. To allow users to filter with other fields such as phone number, address and email.
+
+**4. Allow Copying of Information**
+
+**Currently:** As KnottyPlanners is optimised for keyboard usage, interactions with the GUI through direct copying of information is not supported.
+
+**Plan:** To allow the users of KnottyPlanners to copy information directly from the contact or wedding cards, in order to facilitate mouse usage.
+
+**5. Support for Long Inputs**
+
+**Currently:** KnottyPlanners only support inputs up till a reasonable amount of characters. As such, extreme inputs that exceed our limit can potentially hinder the viewing of information when the user does not resize the window.
+
+**Plan:** To handle extreme inputs of all fields with a long character count, ensuring that the information can still remain visible.
```

Comment:
seems abit vague, was this intentional haha
if not can prob say smth like the ui will show the input completely even if its very long